woocommerce/wc-admin
# Add rating components

Closes #204. 

This PR adds three new components. One of these is a base `Rating` component that allows you to return a set of stars in a filled, empty, or half-filled state. The other two components are `ProductRating` and `ReviewRating` which allow you to pass in product and review API objects in and render the correct rating.

<img width="215" alt="screen shot 2018-07-20 at 3 56 14 pm" src="https://user-images.githubusercontent.com/689165/43022480-766ca8f0-8c35-11e8-9d65-606a35a74e37.png">

To Test:
* Run `npm test` and make sure all tests pass.
* Open the reviews panel and view the examples I've provided.
